docs/asyncapi-file.md

```diff
@@ -1,6 +1,88 @@
 ---
-title: "AsyncAPI Specification File"
-weight: 20
+title: "AsyncAPI document"
```

Comment:
Shouldn't we rename this file to "`asyncapi-document.md`"? 🤔  @Florence-Njeri

Reply:
⬆️  this is the last change I think we need.

@derberg is this ready to be approved/merged by you or..? :O